src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+	 		}
+	 	
+	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--6.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity4;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity4() throws IOException {
+ 	 		
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity2);
+ 	 		Assert.assertTrue(enterQuantity1.isDisplayed());
+ 	 		enterQuantity4.clear();
+ 	 		enterQuantity4.sendKeys(BasePage.getCellData(xlsxName, sheetName, 8, 4));
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--6.quantityAsDecimal"]
+ 	
+ 	 	private WebElement enterQuantity5;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity5() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity2);
+ 	 		Assert.assertTrue(enterQuantity1.isDisplayed());
+ 	 		enterQuantity5.clear();
+ 	 		enterQuantity5.sendKeys("2");
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--12.quantityAsDecimal']")
+ 	 	//*[@id="change--12.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity6;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity6() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity6);
+ 	 		Assert.assertTrue(enterQuantity6.isDisplayed());
+ 	 		JavaScriptExec.sleep();
+ 	 		enterQuantity6.clear();
+ 	 		enterQuantity6.sendKeys("4");
+ 	 		}
+ 	 	
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--6.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity8;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
+ 	 		 * @throws IOException
+ 	 		 */
+ 	 	public void enterQuantity8() throws IOException {
+ 	 		CreateOrderPage sp = new CreateOrderPage(driver);
+ 	 		log.info("Enter Quantity");
+ 	 		JavaScriptExec.sleep();
+ 	 		WaitClass.WaitForElementisDisplay(driver, 10, enterQuantity8);
+ 	 		Assert.assertTrue(enterQuantity8.isDisplayed());
+ 	 		enterQuantity8.clear();
+ 	 		enterQuantity8.sendKeys(BasePage.getCellData(xlsxName, sheetName, 8, 7));
+ 	 		}
+ 	 	
+ 	 	@FindBy(how = How.XPATH, using = "//*[@id='change--6.quantityAsDecimal']")
+ 	 	//*[@id="change--9.quantityAsDecimal"]
+ 	 	private WebElement enterQuantity9;
+
+ 	 		/**
+ 	 		 * Method to Enter Price Rate.
+ 	 		 * 
```

Comment:
@mayankchhajer Please remove the unused xpath

Reply:
Done